routes/v1/readme.md

```diff
@@ -120,6 +120,13 @@ an administrative uid. All other uids passed in will result in an error.
         * `DELETE /:pid`
             * Deletes a post (**Careful**: There is no confirmation!)
             * **Accepts**: No parameters
+        * `POST /:pid/vote`
+            * Votes for a post
+            * **Requires**: `delta`
+            * `delta` must be a number. If `delta > 0`, it's considered an upvote; otherwise, it's a downvote.
```

Comment:
Do you think there are libraries that cannot send `DELETE`s, and only `POST`s? Maybe if it is 0, you can count it as an unvote :stuck_out_tongue: